docs/product/issues/issue-details/replay-issues/index.mdx

```diff
@@ -1,58 +1,21 @@
-- Enable the JavaScript SDK (or framework-specific bundle), version 7.60.1 or higher
+## Quota Consumption
 
 <Note>
-While you can enable **session replay** with JavaScript SDK version 7.27.0, or higher, you'll need to have version 7.60.1 or higher in order to be able to see **rage click issues**.
+Replay issues do not consume issue quota.
 </Note>
 
-## Detection Criteria
-
-"Slow clicks" (also called "dead clicks") are only detected on `<button>`, `<input>`, and `<a>` elements that don't lead to updates to the DOM or a page scroll within 7 seconds). When the user clicks on one of these elements 3 or more times within that 7 second time frame, it indicates frustration, and the SDK registers a "rage click". 
-
-### Why am I seeing too many or too few rage clicks?
-There might be fewer rage clicks than you expect if the user stopped waiting for the site to respond before the 7 second threshold and instead chose to to something else. This is why the rage click issues that you *do* see are so valuable, because the user that clicked at least 3 times and continued waiting at least 7 seconds for the site to respond is likely very frustrated.
-
-You might also see more rage clicks than you expected from buttons that don't trigger a DOM mutation or page scroll (such as Print and Download buttons). There is no reliable way for the SDK to detect that a download or print has initiated, so a slow click might be generated even if the button is not actually "dead". For these cases, you can configure the SDK via `slowClickIgnoreSelectors` - see [Configuration](/platforms/javascript/session-replay/configuration/) for more details.
-
-For example, to ignore detection of dead and rage clicks for download links in your application:
-
-```javascript
-Sentry.replayIntegration({
-  slowClickIgnoreSelectors: [
-    ".download",
-    // Any link with a label including "download" (case-insensitive)
-    'a[label*="download" i]',
-  ],
-});
-```
-
-## Get Rage Click Alerts
-
-To set up alerts and get notified when a rage click occurs, follow these steps:
-
-1. Create a new [Alert Rule](https://sentry.sentry.io/alerts/new/issue/) in Sentry.
-2. In the "Set conditions" section, set the "IF" filter to "The issue's category is equal to", then choose "Replay" from the dropdown.
-3. Add an optional filter if you like.
-5. Choose the action to be performed in the "THEN" dropdown.
-6. Decide how often you'd like Sentry to look for rage click issues.
-7. Lastly, name your alert and add an owner.
+After a replay is accepted, consuming replay quota in the process, error detection takes place and error events are grouped into issues, but without consuming issue quota. These errors are created on the backend after ingest and they will not appear on your [Stats](https://sentry.io/orgredirect/organizations/:orgslug/stats/) page under the "Total Errors" or "Accepted Errors" graphs. You are not billed for their volume.
```

Comment:
The first sentence here doesn't make sense to me. Am I just missing something?

I think the most confusing is the word "accepted", do you mean created? Do our users have to accept something to make a replay actually show up? Who/what is doing the accepting and where and when?

Also, when you say issue quota, do you mean error quota? Just looking at our docs and I don't see "issue quota" https://docs.sentry.io/pricing/quotas/

Maybe instead:
When a replay is created it consumes [replay quota](https://docs.sentry.io/pricing/quotas/manage-replay-quota/) (not to be confused with [error quota](https://docs.sentry.io/pricing/quotas/manage-event-stream-guide/)). After it is created, Sentry checks for errors and any error events associated with the replay are grouped into issues; note that this does _not_ consume issue quota. These errors are created on the backend after ingest and they will not appear on your [stats](https://sentry.io/orgredirect/organizations/:orgslug/stats/) page under the "Total Errors" or "Accepted Errors" graphs. You are not billed for their volume.